src/components/Darkmode/index.tsx

```diff
@@ -0,0 +1,17 @@
+import useTheme from "../../hooks/useTheme";
+
+function Darkmode() {
```

Comment:
una sugerencia en mantener CamelCase
```suggestion
function DarkMode() {
```